src/components/Dropdown/PrioritySelector/__tests__/PrioritySelector.spec.tsx

```diff
@@ -0,0 +1,10 @@
+describe('PrioritySelector', () => {
+    it.todo('renders correctly with all props provided');
```

Comment:
maybe add a test to see that priorities are in the correct order, if you want to control for that (comment above)